engine/src/main/java/org/hibernate/validator/engine/HibernateConstraintViolation.java

```diff
@@ -0,0 +1,7 @@
+package org.hibernate.validator.engine;
+
+import javax.validation.ConstraintViolation;
+
+public interface HibernateConstraintViolation<T> extends ConstraintViolation<T> {
+	Object getInfo();
```

Comment:
yes, this was already mentioned by emmanuel bernard. i propose to have this written in the java doc. WDYT?